modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -172,7 +172,7 @@ var backendServiceInvoker = function () {
         var options = new Array();
         if (IS_OAUTH_ENABLED) {
             var accessToken = getAccessToken();
-            if (!(!accessToken)) {
+            if (accessToken != null || accessToken != "") {
```

Comment:
Using of  if (accessToken) {} was discouraged since accessToken is not a boolean variable.
You can refer to the comment by "prabathabey" above.